5.3/5.3.6/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+	<meta charset="UTF-8">
+	<title>Title</title>
+</head>
+<body>
+### 5.3.6
+Создать объект со следующей логикой:
+Запрашивает у пользователя любой текст: "Введите текст".
+А потом выводит алерт с этим текстом, у которого в каждом слове первая буква - заглавная (большая).
+(Без использования массива)
+<script>
+	const userTextObj = {
+		userText: null,
+		newUserText: '',
+		init() {
+			this.getMessage();
+			this.getFirstBigSymbolWord();
+			this.showMessage();
+		},
+		getMessage() {
```

Comment:
Let's name `askMessage` this method because it doesn't return anything.
If it returns something we will name it `getMessage`